small-talk/components/censorMess.js

```diff
@@ -0,0 +1,11 @@
+import swearWords from './swearWords.js';
+
+export default function censor(input) {
+    const inputinput = input.toLowerCase(); // make sure check for all cases
+    let censored = inputinput;
+    for (let swearWord of swearWords) { // check for all swear words
+        const regex = new RegExp(`\\b${swearWord}\\b`, 'gi'); // used AI to match words and replace
```

Comment:
I believe with the 'i' flag here, you wouldn't need a call toLowerCase, since it should ignore case sensitivity in the regex expression.

Reply:
Thank you for pointing that out. I corrected it and tested that it works